tools/daemon/daemon.h

```diff
@@ -67,7 +68,7 @@
 #include "core/util/list.h"
 #include "utils/clock.h"
 
-#define MODULE_NAME "xliod"
+#define MODULE_NAME "xliod: "
```

Comment:
why in some places you remove and some you add?

Reply:
just means my regex games weren't good enough :)

It was inconsistent before so I allowed myself to keep it that way. could revert.

Reply:
: